### ISSUE TITLE
Slow scan but fast settings and good connection. 

### ISSUE THREAD

**Issue by KeooR** [Maxx53/SCMBot#449]
Hello, my scan is pretty slow and I do not know why... 
![why](https://cloud.githubusercontent.com/assets/13817999/9605441/064ee566-50be-11e5-85a2-c92982990069.png)
Can you explain ? Thanks !

**Comment by Randjolaf** [comment]
this happens to me as well. And someone keeps buying the items I find! I keep getting the message "you cannot purchase this item, because someone else has already purchased it"! What are those other people using then???

**Comment by mtzi** [comment]
@KeooR its probably because of many items, i guess u have 5-6 items on these settings?

@Randjolaf you must have cookie problem, did u insert all cookies? if you can, try to buy something very cheap (0.03) on normal price sometime to see if it can buy at all or if u have problem

**Comment by KeooR** [comment]
@mtzi No. Only one. ( You can see it on my screenshot ).

**Comment by Randjolaf** [comment]
@mtzi I did buy something very cheap, but only once and it wasn't a popular item. Even I hate it now xD And once I bought an item that cost 10 euros. But I am not sure if I have deleted and reinstalled the program because sometimes it gets completely bugged. So the cookies aren't in the program now. And I have a big problem with that as I can't find 2 cookies at all now! They don't exist! SteamLoginSecure and SteamMachineAuth don't exist! 
btw, here is a screenshot of what happens to me:
![scmbot bug](https://cloud.githubusercontent.com/assets/13485669/9638495/65594e86-51a7-11e5-916c-896f1d115fad.png)

**Comment by nas-** [comment]
@Randjolaf 
And I have a big problem with that as I can't find 2 cookies at all now! They don't exist! SteamLoginSecure and SteamMachineAuth don't exist! 

You have to go to httpS://steamcommunity.com/market
The S of httpS is important

**Comment by mtzi** [comment]
@Randjolaf if u cant find these 2 cookies u cant buy, i think try what @nas- said. also, for ti5 immortals because there is 10000000 bots on them i dont think u can buy all, but you should get some. good luck

**Comment by KeooR** [comment]
If you use Opera you have no problems to importe cookies. ( On chrome a kind of cookies are hidden )

**Comment by radam9** [comment]
on google chrome you have to go to:>

Chrome://Settings/Cookies ---> search for "Steam" ---> Click on "Steamcommunity.com"

then click on the items u want the values for, you will find all of them there.

**Comment by KeooR** [comment]
@mtzi I have and other question. For example : If i have " 0 " delay between requests/ " 0 " in delay, but i have a response from the server more than 1 seconde this is risky ? 
![botaaa](https://cloud.githubusercontent.com/assets/13817999/10264314/fce09610-6a07-11e5-8e17-57f234247c45.png)

**Comment by mtzi** [comment]
I dont know... never tried... but since u got ping ever 3-4 sec, why not make delay 2000? it brings same result

**Comment by KeooR** [comment]
@mtzi because i want to have a ping every 1 sec :/ So i should set a bigger delay ?

**Comment by mtzi** [comment]
no, i dont know how to get ping every 1 sec, really... my best with 600 delay at some point was 3 pings every 4 sec, which is almost every 1, anyway, u dont need that fast scan on items lke those listed though, u buy these items with 3sec scans too. if u want scan every one put 1 or 2 items with 600 delay, that should do it

**Comment by KeooR** [comment]
@mtzi ok thanks
